.dev/scripts/copyRuntime.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+source .env
+CONNECTORPATH=$(pwd)
```

Comment:
The connector path would be 

```sh
# full path stripped of three path segments
CONNECTORPATH="$(dirname "$(dirname "$(dirname "$(realpath "$0")")")")"
```

`$(pwd)` only works from inside the connector's root directory.
```